spec.html

```diff
@@ -48676,24 +48676,19 @@ <h1>
         <emu-alg>
           1. Assert: _generator_.[[AsyncGeneratorState]] is ~draining-queue~.
           1. Let _queue_ be _generator_.[[AsyncGeneratorQueue]].
-          1. If _queue_ is empty, then
-            1. Set _generator_.[[AsyncGeneratorState]] to ~completed~.
-            1. Return ~unused~.
-          1. Let _done_ be *false*.
-          1. Repeat, while _done_ is *false*,
+          1. Repeat,
+            1. If _queue_ is empty, then
```

Comment:
I guess this could also be written `Repeat, while _queue_ is not empty` and then put this logic at the end, which would maybe match the way it would be written in an implementation better? Either's fine by me.